corehq/apps/export/static/export/js/list_exports.ng.js

```diff
@@ -80,6 +80,11 @@
             var currentUrl = useLegacyBulkExportUrl ? $scope.legacy_bulk_download_url : $scope.bulk_download_url;
             input.closest("form").attr("action", currentUrl);
         };
+        $scope.downloadRequested = function($event) {
+            var $btn = $($event.target);
+            $btn.addClass('disabled');
```

Comment:
it's my claim to fame on stackoverflow: http://stackoverflow.com/questions/10867506/dragleave-of-parent-element-fires-when-dragging-over-children-elements/14027995#14027995 😄